test/core/loop.wast

```diff
@@ -765,6 +765,26 @@
   ))
   "type mismatch"
 )
+(assert_invalid
+  (module (func $type-param-brif-void-vs-num
+        (f32.const 0)
+        (loop (param i32)
```

Comment:
Here the error is that the param doesn't match the operand type, the br_if isn't the primary problem.

Nit: stick to +2 indentation